docs/source/index.rst

```diff
@@ -117,6 +117,35 @@ One of the reasons DFS is so powerful is that it can create a feature matrix for
     feature_matrix_sessions.head(5)
 
 
+Understanding Feature Output
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. ipython:: python
+    :suppress:
+
+    features = {f.get_name(): f for f in features_defs}
```

Comment:
for continuity reasons, what if we accessed `feature_defs` directly.  something like

```
>  example_feature = feature_defs[11]
> print(example_feature)
> graph_feature(example_feature)
```